homework/readme.md

```diff
@@ -15,6 +15,7 @@
 | Date Assigned | Due Date | Assignment |
 |---|---|---|
 | Week 6 | Tuesday | [Draft of Resume](https://github.com/ga-dc/outcomes-resume) (Outcomes)|
```

Comment:
Correct hw link for resume is https://docs.google.com/spreadsheets/d/1qqP9H_CK-WYRyzksaaF-Nk9GEKF_OEAtomICdlJoN6I/edit#gid=1038012474 

This will probs fix the merge conflict :smile: